### PR TITLE
Support set visualised pairs in strategy

### PR DESCRIPTION
**Pull request by hieuh25**
- Add support for `state.visualisation.set_visualised_pairs()` to override default trading pairs which are used for price chart
- Save pair IDs in `state.visualisation.pair_ids`

### CODE REVIEW

File: tradeexecutor/state/visualisation.py [hunk 1]
```diff
@@ -358,10 +359,30 @@ class Visualisation:
...
+    def set_visualised_pairs(self, pairs: list[TradingPairIdentifier | DEXPair]) -> None:
+        """Set the trading pair for this plot.
+
+        :param pairs:
+            Use these trading pairs to plot candle charts instead of the default
+            trading pairs from strategy universe.
+        """
+        assert isinstance(pairs, list)
+        for pair in pairs:
+            if isinstance(pair, DEXPair):
```
Comment (miohtama) [inline tradeexecutor/state/visualisation.py:380]: DEXPair should never happen here, as new strategy functions always return TradingPairIdentifier.

If they do not, the pair can be converted earlier.

File: tests/test_strategy_state_visualisation_multipair.py [hunk 2]
```diff
@@ -386,3 +393,60 @@ def test_visualise_strategy_state(
...
+        reserve_currency=ReserveCurrency.busd,
+        trade_routing=TradeRouting.user_supplied_routing_model,
+        routing_model=routing_model,
+        log_level=logging.WARNING,
+        allow_missing_fees=True,
+    )
+
+    # visualise only one pair
+    state.visualisation.set_visualised_pairs([
+        strategy_universe.data_universe.pairs.get_pair_by_id(99),
```
Comment (miohtama) [inline tests/test_strategy_state_visualisation_multipair.py:425]: You can use `strategy_universe.get_pair_by_human_description` and it returns `TradingPairIdentifier` directly

**Review by miohtama — CHANGES_REQUESTED** [review]

File: tradeexecutor/state/visualisation.py (hunk 1 above)
Comment (hieuh25) [reply tradeexecutor/state/visualisation.py]: Fixed

File: tests/test_strategy_state_visualisation_multipair.py (hunk 2 above)
Comment (hieuh25) [reply tests/test_strategy_state_visualisation_multipair.py]: `get_pair_by_human_description()` actually returns `DEXPair` as well 😂, but I got passed it with `get_trading_pair()`

**Pull request merged**